stograde/process_assignment/Analysis_Result.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+from stograde.process_assignment.Assignment_Status import AssignmentStatus
```

Comment:
You can use relative imports for these: "from .assignment_status import AssignmentStatus"

Reply:
That's probably there because that's how auto import puts it. I'll go back and make sure to change those.

Reply:
Makes sense. I'll just keep an eye out for them when I do a final review.